packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getRootOverflows.st

```diff
@@ -0,0 +1,6 @@
+information extraction
+getRootOverflows
+	
+	self messageTally gcStats ifNil: [^ nil].
+	
+	^ self messageTally gcStats at: 22
```

Comment:
Please add a function `getGcStats`, as mentioned earlier to avoid all this code duplication caused by the many uses of `self messageTally gcStats ... `.